tests/nlu/base/test_utils.py

```diff
@@ -61,42 +76,30 @@ def test_ordered():
     assert ordered(target) == [("a", [1, 2, 3]), ("b", 1), ("c", "a")]
 
 
-@pytest.mark.parametrize(
-    ("model_dir", "expected"),
-    [
-        ("test_models/test_model_mitie/model_20170628-002704", True),
-        ("test_models/test_model_mitie_sklearn/model_20170628-002712", True),
-        ("test_models/test_model_spacy_sklearn/model_20170628-002705", True),
-        ("test_models/", False),
-        ("test_models/nonexistent_for_sure_123", False),
-    ],
-)
-def test_is_model_dir(model_dir, expected):
-    assert is_model_dir(model_dir) == expected
+def test_empty_is_model_dir(empty_model_dir):
```

Comment:
I know an empty directory passes ` specifically checks if the directory has no subdirectories and if all files have an appropriate ending` but why do we want a empty directory to pass as a model directory?

Reply:
The check is in there to avoid deleting files we don't want to delete. If it is empty 🤷‍♂ 

Can happen if persist fails.